index.js

```diff
@@ -0,0 +1,330 @@
+let playerOne = '';
+let playerTwo = '';
+let clockCount = 0;
+let interval;
+let timer = 0;
+
+// initializing variables with DOM scripting
+let startBtn = document.getElementById('start');
+startBtn.disabled = true;
+let onePlayer = document.getElementById('onePlayer');
+let twoPlayer = document.getElementById('twoPlayer');
+let playerOneName = document.getElementById('playerOne')
+let playerTwoName = document.getElementById('playerTwo')
+let currentPlayer = 'x';
+let status = document.getElementById('status-bar');
+let clock = document.getElementById('clock');
+let cells = Array.from(document.getElementsByTagName('td'));
+let cellOne = document.getElementById('cell-1')
+let cellTwo = document.getElementById('cell-2')
+let cellThree = document.getElementById('cell-3')
+let cellFour = document.getElementById('cell-4')
+let cellFive = document.getElementById('cell-5')
+let cellSix = document.getElementById('cell-6')
+let cellSeven = document.getElementById('cell-7')
+let cellEight = document.getElementById('cell-8')
+let cellNine = document.getElementById('cell-9')
+let usedCellArray = [];
+let gameMode = '';
+
+function capitalize(string) {
+    string = string[0].toUpperCase() + string.slice(1).toLowerCase();
+    return string;
+}
+
+function initialize() {
+
+}
+
+twoPlayer.addEventListener('mouseover', (event) => {
+  event.target.style.color = '#42aff7'
+})
+
+twoPlayer.addEventListener('mouseout', (event) => {
+  event.target.style.color = 'black'
+})
+
+onePlayer.addEventListener('mouseover', (event) => {
+  event.target.style.color = '#42aff7'
+})
+
+onePlayer.addEventListener('mouseout', (event) => {
+  event.target.style.color = 'black'
+})
+
+startBtn.addEventListener('mouseover', (event) => {
+  event.target.style.color = '#42aff7'
+})
+
+startBtn.addEventListener('mouseout', (event) => {
+  event.target.style.color = '42aff7'
+})
+
+
+// assigning individual cells on your game board
+// possible clicks
+let cellArray = [
+    cellOne,
+    cellTwo,
+    cellThree,
+    cellFour,
+    cellFive,
+    cellSix,
+    cellSeven,
+    cellEight,
+    cellNine
+]
+
+// all the possible combinations to win the game
+let WINNING_COMBOS = {
+    rowOne: [cellOne, cellTwo, cellThree],
+    rowTwo: [cellFour, cellFive, cellSix],
+    rowThree: [cellSeven, cellEight, cellNine],
+    columnOne: [cellOne, cellFour, cellSeven],
+    columnTwo: [cellTwo, cellFive, cellEight],
+    columnThree: [cellThree, cellSix, cellNine],
+    forwardSlash: [cellThree, cellFive, cellSeven],
+    backSlash: [cellOne, cellFive, cellNine]
+}
+
+twoPlayer.addEventListener('click', () => {
+    twoPlayer.disabled = true;
+    onePlayer.disabled = true;
+    startBtn.disabled = false;
+    gameMode = 'twoPlayer';
+})
+
+onePlayer.addEventListener('click', () => {
+    twoPlayer.disabled = true;
+    onePlayer.disabled = true;
+    startBtn.disabled = false;
+    gameMode = 'onePlayer'
+})
+
+// startColors();
+
+cellOne.addEventListener('click', startColors)
```

Comment:
This could be more practical to do on clicking start, rather then on cell one, but you could also add a hover effect on the squares using CSS and not need to write out all of these functions